_posts/2022-02-01-newpipe-0.21.16-hotfix.md

```diff
@@ -0,0 +1,50 @@
+﻿---
+layout: post
+title: "NewPipe 0.21.16 released: Do not panic! We fixed YouTube."
+short: "NewPipe 0.21.16 released"
+date: 2022-02-01 09:00:00 + 01:00
+categories: [pinned, release]
+author: <a href="https://github.com/poolitzer">@Poolitzer</a>
+image: newpipe
+excerpt_separator: <!-- more -->
+---
+
+Hello, did you order a hot fix update? Because we have one here, it is steaming and ready.
+
+<!-- more -->
+
+## YouTube broke.
+
+We are very sorry about this, but YouTube changed a parameter in their video player decryption and the code we created to decrypt the input from YouTube broke. [@AbduAmeen](https://github.com/AbduAmeen) created a fix together with [@SebastianSong](https://github.com/SebastianSong), and [@TobiGr](https://github.com/TobiGr) worked with them to get it merged in the Extractor. Thanks to all three of them for doing this so quickly.
+
+## Quick note for the next time this happens
+
+We, as every development team, feel great knowing our app is used by many people. We mostly notice this when it breaks in a major way, of course. This time it was a bit too much though. We received 74 comments in 10 hours (!) under the one thread for this issue, 18 duplicate threads (!!) of this same issue and quite some comments on various (!!!) blog posts regarding NewPipe being broken.
+
+The problem with having that many comments is that the development team needs specific information to figure out the cause of the issue. If so many messages are send to us, we are potentially missing the one important hint to find the cause. This means it slows down the development of the fix, since every single message might need to be checked and new ones keep coming in when they aren't needed.
```

Comment:
I think it should be sent
```suggestion
The problem with having that many comments is that the development team needs specific information to figure out the cause of the issue. If so many messages are sent to us, we are potentially missing the one important hint to find the cause. This means it slows down the development of the fix, since every single message might need to be checked and new ones keep coming in when they aren't needed.
```

Reply:
oh right.